app/uk/gov/hmrc/fhregistrationfrontend/views/EditMode.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.fhregistrationfrontend.views
+
+object EditMode extends Enumeration {
```

Comment:
EditMode looks like it should be named just Mode?

Reply:
this is something we thought about and it only relates to how users can edit the data, I can change throughout if you like?

Reply:
I have renamed the object and params to Mode